templates/admin/view_rta_regenerate.php

```diff
@@ -70,6 +70,9 @@
         </div>
       </div>
 
+        <a href="https://help.shortpixel.com/article/233-quick-guide-to-using-regenerate-thumbnails-advanced-settings"
+           class="shortpixel-help-link" target="_blank">
```

Comment:
the class `shortpixel-help-link` isn't actually needed because it is not defined, so you can safely remove it.